stanford/stage/group_vars/all

```diff
@@ -25,9 +25,12 @@ EDXAPP_FEATURES_ENABLE_SHOPPING_CART: true
 EDXAPP_FEATURES_USE_MICROSITES: true
 EDXAPP_LMS_ENABLE_BASIC_AUTH: True
 EDXAPP_LMS_NGINX_PORT: 80
+EDXAPP_PREVIEW_LMS_BASE: "preview.{{ EDXAPP_LMS_SITE_NAME }}"
```

Comment:
For variables like this which are identical across all 4 instances, can they be factored out into a separate file, or does that not work for Ansible group_vars? Vars like COMMON_DEPLOYMENT_DIR, COMMON_HOSTNAME, EDXAPP_CMS_NGINX_PORT, EDXAPP_LMS_NGINX_PORT, etc.

Reply:
Yeah, I'd like to do that eventually. The easiest thing I've thought of would be to symlink a common file between `group_vars` directories, but that requires splitting some files out and I opted to defer that work for the time being.

Note: we can't set these vars directly in the roles' `defaults/main.yml` files because we _don't_ want them overridden for devstack, etc.